include/podio/utilities/TypeHelpers.h

```diff
@@ -37,19 +30,15 @@ namespace det {
     void operator=(const nonesuch&) = delete;
   };
 
-  template <template <typename...> typename Op, typename... Args>
-  using is_detected = typename detail::detector<nonesuch, void, Op, Args...>::value_t;
-
-  template <template <typename...> typename Op, typename... Args>
-  static constexpr bool is_detected_v = is_detected<Op, Args...>::value;
-
   template <typename DefT, template <typename...> typename Op, typename... Args>
   using detected_or = detail::detector<DefT, void, Op, Args...>;
 
   template <template <typename...> typename Op, typename... Args>
-  using detected_t = typename detail::detector<nonesuch, void, Op, Args...>::type;
```

Comment:
This is removed, because it's unused, I suppose?

Reply:
Yes, only used in `GetCollectionType` that is also removed because it's unused.